scripts/common.js

```diff
@@ -150,13 +150,14 @@ async function openExtensionTab(url) {
 
 async function openTab(tab, windowId, automatic = false) {
 	var windows = await getAllWindows();
+	var cookieStoreId = tab.cookieStoreId ? { cookieStoreId: tab.cookieStoreId } : {};
 	if (tab.incognito) {
 		var w = windows.find(i => i.incognito) || await createWindow(undefined, t.incognito);
-		await new Promise(r => chrome.tabs.create({url: tab.url, active: false, pinned: tab.pinned, windowId: w.id}, r));
+		await new Promise(r => chrome.tabs.create({url: tab.url, active: false, pinned: tab.pinned, ...cookieStoreId, windowId: w.id}, r));
 	} else if (!windows || !windows.filter(w => !w.incognito).length) {
-		await new Promise(r => chrome.windows.create({url: tab.url}, r));
+		await new Promise(r => chrome.windows.create({url: tab.url, ...cookieStoreId}, r));
```

Comment:
`windows.create` takes the optional `cookieStoreId` parameter in Firefox ([source](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/create#cookiestoreid)).